src/cli.js

```diff
@@ -58,29 +68,35 @@ commander
   .description("start everything")
   .option("-T, --no_tests", "ignore test hook")
   .option(debugOption.command, debugOption.description)
-  .action((options) => startAll(options.no_tests, options.debug));
+  .action((options) => startAll(options.no_tests, options.debug))
+  .action((options)=> updateLastVersionUsed());
 
 commander
   .command("build")
   .description("create production asset build")
-  .action(() => startClient(true));
+  .action(() => startClient(true))
+  .action((options)=> updateLastVersionUsed());
```

Comment:
I believe eslint is going to complain that `options` is not used. Since we aren't using it we don't need to define the arg. This goes for all of the similar lines like this below